pkg/feeds/getter.go

```diff
@@ -17,7 +17,7 @@ import (
 
 // Lookup is the interface for time based feed lookup
 type Lookup interface {
-	At(ctx context.Context, at, after int64) (swarm.Chunk, error)
+	At(ctx context.Context, at, after int64) (chunk swarm.Chunk, currentIndex, nextIndex Index, err error)
```

Comment:
TODO: in fact At when used with noncurrent time should not return a next index.
only  latest  should..